src/main/resources/application.yaml

```diff
@@ -18,6 +24,13 @@ queue:
 
 idam:
   s2s-auth:
-    url:    ${S2S_URL:http://localhost:8081}
+    url:    ${S2S_URL:http://localhost:4552}
     secret: ${S2S_SECRET:AAAAAAAAAAAAAAAA}
     name:   ${S2S_NAME:bulk_scan_orchestrator}
+  api:
+    url: http://localhost:8080
+  client:
+    id: 'bsp'
+    secret: '123456'
+    redirect_uri: 'http://localhost/receiver'
+  users:
```

Comment:
I assume spring will magically create a map here based on env vars following a specific convention.
May be worth adding a comment here.

Reply:
yes, need to look at the s2s code to get that bit fully working, currently its not

Reply:
It works now, and infra config added here:
https://github.com/hmcts/bulk-scan-orchestrator/pull/37#pullrequestreview-157571905
with a comment in that PR